README.md

```diff
@@ -134,7 +128,7 @@ kubectl apply -f https://raw.githubusercontent.com/cnoe-io/backstage-terraform-i
             - allow: [User, Group]
 ```
 
-8. Finally, run the `idpbuilder` command `idpbuilder create --use-path-routing --package-dir examples/ref-implementation -n` to incrementally build and run the terraform backstage integrations.
+7. Finally, run the `idpbuilder` command `idpbuilder create --use-path-routing --package-dir examples/ref-implementation --package-dir examples/terraform-integrations -n` to incrementally build and run the terraform backstage integrations.
```

Comment:
I dont think you want to set the `-n` flag here. using `no-exit` is more of an advanced usecase that we dont want the users to necessarily start with.

Reply:
@nimakaviani  Fixed it.